Services/CO.CDP.OrganisationInformation.Persistence/DatabaseOrganisationJoinRequestRepository.cs

```diff
@@ -33,6 +33,12 @@ public async Task<IEnumerable<OrganisationJoinRequest>> FindByOrganisation(Guid
             .ToArrayAsync();
     }
 
+    public async Task<OrganisationJoinRequest?> FindByOrganisationAndPerson(Guid organisationId, int personId)
```

Comment:
We use `Guid` for organisation ID, while `int` for person id. Any particular reason for this?

Reply:
No reason, updated to Guid.